### DISCUSSION TITLE
Please add slot machines, blackjack, poker to the outposts

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#14247] [Ideas and suggestions]
Please add slot machines, blackjack, poker to the game. Let them be at the stations so that the crew does not get bored while the captain is preparing a trip to sea.

Lets our outposts to be "with blackjack and hookers"

**Comment by Zimon3283** [comment]
Rather any card game where a round takes quite a short time so you have something to do while waiting for the captain to take a shit.

**Comment by BlackMoons** [comment]
Honestly I am all for this, we need some 5 minute activities to do on the outposts to keep idle hands from becoming the devils plaything. 

Any time someones fabricating or doing other outpost stuff, everyone always gets bored as hell and boredom leads to griefing.

**Comment by pavel4win** [comment]
Gambling will help the crew spend their salaries. At present, the crew simply has nowhere to spend their salaries.
